### ISSUE TITLE
Unshared records: Explore page and individual record view

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1765]
This relates to https://irecord.org.uk/explore/my-unshared

At the moment we have buttons to view the record, and to view the species details page. The latter is arguably not needed here, but does no harm so happy to keep it.

The button to view the record takes you to a page with URL starting "irecord.org.uk/record-details/verification" - is this called a "verification" view because it is only accessible to verifiers?

**Editing/deleting an unshared record**
We will need to add an edit option for the individual records, probably via buttons on both the Explore page and on the /record-details/verification page. Or maybe this can be tackled using the same functionality as #1673?

**Verification**
As intended, the unshared records appear in the verification page for those verifiers that have filters matching the unshared data. From the verifier's page, it is possible to edit the record via the edit button, which is fine. The button to view the record currently produces "The record cannot be found". I guess the ideal solution for the verifiers' 'view record' button would be for it to go to the /record-details/verification view when unshared records are being looked at. However, if that is complex to code in then I think we can live without it.

**Comment by johnvanbreda** [comment]
>  is this called a "verification" view because it is only accessible to verifiers?

Yes, it differs from the other version in that it allows viewing of any record that is available for verification on iRecord rather than being limited to websites that are available for reporting.

> We will need to add an edit option for the individual records,

What would be the use case here? Would a bulk edit (limited to just a few fields) be OK or do we need to support individual records? I would have thought a a link to the generic edit form would be a good start?

> I guess the ideal solution for the verifiers' 'view record' button would be for it to go to the /record-details/verification view when unshared records are being looked at. However, if that is complex to code in then I think we can live without it.

Would it be acceptable for the verification pages to always link to record-details/verification page - it can display any verification record including unshared, so is not just limited to unshared.

**Comment by kitenetter** [comment]
Edit option: the ideal would be for a full edit option to be available for each record individually, using the generic editing form, and for the bulk editing tools to be available for groups of the unshared records.

Viewing records: I think you are suggesting that on the verification page the "view record" button should **always** go to a record-details/verification view? If so I can't see any problem with that, and I agree it would be a good way of making records visible to verifiers that can't be made public in iRecord.

**Comment by johnvanbreda** [comment]
All done
